src/EnergyPlus/UnitarySystem.cc

```diff
@@ -7222,6 +7254,45 @@ namespace UnitarySystems {
 
                 thisSys.processInputSpec(state, input_specs, sysNum, errorsFound, ZoneEquipment, ZoneOAUnitNum);
 
+                // heat recovery loop inputs
```

Comment:
Read the 3 new inputs in CoilSystem:Cooling:Water that define heat recovery loops.